doc/api/errors.md

```diff
@@ -1809,6 +1809,14 @@ recommended to use 2048 bits or larger for stronger security.
 A TLS/SSL handshake timed out. In this case, the server must also abort the
 connection.
 
+<a id="ERR_TLS_INVALID_CONTEXT">
+### ERR_TLS_INVALID_CONTEXT
+<!-- YAML
+added: REPLACEME
+-->
+
+The context must be a `NativeSecureContext`.
```

Comment:
So … I realize this is a it confusing, but:

- The `.context` property of the `context` variable is not public API, despite its public-sounding name. We should not be referring to it in the message. (There are also [old comments](https://github.com/nodejs/node/blob/84a25eb7949a8ea887ddf8e23aa30f23a72fc5b1/lib/_tls_wrap.js#L188) explicitly mentioning that this should not be part of the public API.)
 - `NativeSecureContext` isn’t the class name, only the variable name in the tls code, to avoid confusion with the public JS API class named `SecureContext`.

I’d go with `SecureContext` rather than `NativeSecureContext` and `context` rather than `context.context` in the documentation and the messages. `context.context instanceof NativeSecureContext` is essentially only a stricter brand check than `context instanceof SecureContext`.

Reply:
Thanks for taking the time to explain. Updated.